projects/container_bench/testing/container_engine.py

```
@@ -178,6 +213,51 @@ def init(self):
+    Write-Output "Still waiting... ($elapsed/$timeout seconds)"
+    Start-Sleep -Seconds $interval
+    $elapsed += $interval
+}}
+
+if ($elapsed -ge $timeout) {{
+    Write-Output "Warning: Timeout waiting for podman machine to be ready"
+    type $env:USERPROFILE\podman_script_log.txt
+}}
+
+Remove-Item "$env:USERPROFILE\start_podman.ps1" -Force -ErrorAction SilentlyContinue
+"""  # noqa: E501
```

Ruff (0.13.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)